homeassistant/components/deluge/coordinator.py

```diff
@@ -44,7 +44,7 @@ async def _async_update_data(self) -> dict[Platform, dict[str, Any]]:
             _data = await self.hass.async_add_executor_job(
                 self.api.call,
                 "core.get_session_status",
-                DATA_KEYS,
+                [iter_member.value for iter_member in list(DelugeGetSessionStatusKeys)],
```

Comment:
Might work to do list(DelugeGetSessionStatusKeys) when you use StrEnum instead.

Reply:
see above comment, deluge-client doesn't seem to like it it if `DelugeRPCClient.call` has non primitive arguments